testing/dart/fragment_shader_test.dart

```diff
@@ -402,22 +402,23 @@ void main() async {
     path.join('supported_glsl_op_shaders', 'iplr'),
     '.iplr',
   );
-  expect(iplrSupportedGLSLOpShaders.isNotEmpty, true);
```

Comment:
Why are we losing these asserts?  I see the print statement below, are we just going to fail further down in execution?

Reply:
Package test can't run assertions outside of a test block and these functions generate tests. I think the test will still fail approximately at the same time though?

Reply:
Changed to a `fail`